Replicator/ChangesFeed.cc

```diff
@@ -47,8 +47,10 @@ namespace litecore { namespace repl {
     ,_skipDeleted(_options->skipDeleted())
     {
         DebugAssert(_checkpointer);
-        auto i = _options->collectionSpecToIndex().at(_checkpointer->collection()->getSpec());
-        _continuous = _options->push((CollectionIndex)i) == kC4Continuous;
+
+        // JIM: This breaks tons of encapsulation, and should be reworked
```

Comment:
In a working session, collectionIndex and collectionSpec are synonyms: they should refer to the same thing. Most places we use the index, for it's faster. Very few places we use the collectionSpec.